psqlgraph/query.py

```diff
@@ -303,7 +305,7 @@ def subq_path(self, path, filters=None, __recurse_level=0):
         edge, this_id, next_id, target_class = link_details
 
         # Construct the next recursive level's base query and recurse
-        next_node_q = self.session.query(target_class)
+        next_node_q = self.session.query(target_class, package_namespace=self.package_namespace)
```

Comment:
I never tested this behavior, but what's going to happen if we pass in a generic `Session` object into `with graph.session_scope(session=nonGraphSessionInstance):`? and do we want to make sure we don't accidentally mess with something that we don't want to?

Reply:
for this particular line of code, it will just be a part of kwargs and not get used. Generally, I think passing a `nonGraphSessionInstance` will definitely cause lots of hiccups in the usage of the library as the `GraphSession` returns a `GraphQuery` which allows us do extra querying like `.paths`. Passing a different session instance will return a plain sqlalchemy session, unless the person does something extra to make it return a `GraphQuery`